### ISSUE TITLE
Package for Archlinux User Repository (AUR)

### ISSUE THREAD

**Issue by tijko** [kinnay/netlink#2]
I was going to volunteer for maintaining the `PKGBUILD` for AUR.  Currently using `pip install` is not allowed, unless flagging with `--break-system-packages`, which is not ideal.

This is a very straightforward task and would provide users of Archlinux a conformant method of installation.

**Comment by kinnay** [comment]
I'm not against that, if it doesn't require any effort from my side.

I'm wondering about the benefits though. Would such a package be mostly for convenience? The recommended way seems to be to install packages in a virtual environment instead, but I'm not familiar with Arch myself.

**Comment by tijko** [comment]
> if it doesn't require any effort from my side.

Not at all, I was tossing my hat in the ring.  I've followed `libnl` and have written a decent amount of C level Netlink code.  having this project was something that I thought would benefit the community.  Which ties in to your next bit:

> I'm wondering about the benefits though.  Would such a package be mostly for convenience?

Thats pretty much it.  You are able to install with `yay` or any other Archlinux package tools.
